src/schemas/ba-sak/begrunnelse/EØSTriggere/annenForeldersAktivitetTrigger.ts

```diff
@@ -0,0 +1,43 @@
+import { EØSBegrunnelseDokumentNavn, SanityTyper } from '../../../../util/typer';
+import { erEøsBegrunnelse, hentEØSTriggereRegler } from './utlis';
```

Comment:
Fikser du at filen vi importerer fra her skal heter `utils.ts` i stedet for `utlis.ts`?